README.md

```diff
@@ -1,26 +1,255 @@
 # JPL Open Source Rover Code
```

Comment:
This readme has become rather large. I personally favor succinct readmes with references to other markdown files. A user is ideally only going to look at installation once and then will want to move on to look at different parts of the readme such as how to configure the `yaml` files, how to do the startup service (which I only set up a few months after installing), and how the code works.

Reply:
Indeed, good thought. I'll figure out how to break this up in a better way. Up to now the effort has been focused on just getting this on the page.

Reply:
Broken up now.